eva-lib/src/main/java/uk/ac/ebi/eva/lib/repository/VariantEntityRepositoryImpl.java

```diff
@@ -81,6 +81,12 @@ public Long countByRegionsAndComplexFilters(List<Region> regions, List<VariantEn
         return countByComplexFiltersHelper(query, filters);
     }
 
+    @Override
+    public List<String> findDistinctChromosomes() {
+        return (List<String>) mongoTemplate.getCollection(mongoTemplate.getCollectionName(VariantEntity.class))
+                                           .distinct("chr", new Query().getQueryObject());
```

Comment:
It shouldn't be necessary to create an object for an empty query (at least the low level API allows to indicate only the distinct field).